estate/models/estate_property_offer.py

```diff
@@ -36,11 +36,12 @@ def _inverse_date_deadline(self):
     def create(self, vals_list):
         for vals in vals_list:
             property = self.env['estate.property'].browse(vals['property_id'])
-            if property.state == 'new':
-                property.state = 'offer_received'
+            if property:
+                if property.state == 'new':
+                    property.state = 'offer_received'
```

Comment:
instead of indenting everything I would do:
```suggestion
           if not property:
                 raise UserError(_("Problem"))
              if property.state == 'new':
                    property.state = 'offer_received'
```